main.tex

```diff
@@ -774,22 +790,21 @@ \subsubsection{Support for table concatenation}
 tables have the same columns, we may want to stack them ``vertically'' to create a
 new table with the same columns but all rows. If two tables are row-matched but
 have distinct columns, we may want to stack them ``horizontally'' to create a
-new table with the same rows but all columns. For other situations, more general
-table concatenations or joins are also possible when two tables share some
```

Comment:
Note: "join" is a verb, so I used "joining", the noun counterpart. I thought of using "joint" but I don't think that is correct.

Reply:
I think in the database world, a join can be a noun: https://en.wiktionary.org/wiki/join#Noun - I like it better as it was (not the gerund)